docs/reference/server-configuration.md

```diff
@@ -145,19 +167,23 @@ For more information, see
 
 ## `ksql.functions.substring.legacy.args`
```

Comment:
What is this config? Interesting that it already had a note about being per-query, if so then we would need to include it in the query-level properties list here but I actually can't seem to find any config with this or even a related name in the current codebase. I'm guessing it was deprecated/removed at some point? Should we also remove it from the docs here? cc @MichaelDrogalis who added the docs for it

Reply:
I removed this puppy.